dss/events/handlers/notify_v2.py

```diff
@@ -220,6 +222,19 @@ def build_deleted_bundle_metadata_document(key: str) -> dict:
         "version": version,
     }
 
+def _dot_to_underscore_and_strip_numeric_suffix(name: str) -> str:
```

Comment:
LGTM: is the modification for the name due to jmespath allowing [SubExpressions](http://jmespath.org/specification.html#subexpressions)?

Reply:
Yes, this change makes it easier to use JMESPath sub-expressions. It was possible before but the syntax is unfortunate.

The second reason is this makes metadata file names the same in ES and JMESPath index documents.